### PR TITLE
PSR12 and Object Calisthenics

### PR DESCRIPTION
**Pull request by viniciuseneas**
PSR12 - Blank lines may be added to improve readability and to indicate related blocks of code
Object Calisthenics - Don’t use “Else” keyword

empty - Determines whether the variable is empty or null.

### CODE REVIEW

**Comment by viniciuseneas** [comment]
Ok, I understand your point. The comparison is more performative.

File: classes/phing/listener/DefaultLogger.php [hunk 1]
```diff
@@ -159,20 +159,22 @@ public function buildStarted(BuildEvent $event)
...
+
         $msg .= PHP_EOL . "Total time: " . self::formatTime(Phing::currentTimeMillis() - $this->startTime) . PHP_EOL;
 
         if ($error === null) {
             $this->printMessage($msg, $this->out, Project::MSG_VERBOSE);
-        } else {
-            $this->printMessage($msg, $this->err, Project::MSG_ERR);
         }
+
+        $this->printMessage($msg, $this->err, Project::MSG_ERR);
```
Comment (siad007) [inline classes/phing/listener/DefaultLogger.php]: This will **always** print an error message!

Comment (viniciuseneas) [reply classes/phing/listener/DefaultLogger.php]: OMG! Really! I will put a ternary.

Comment (siad007) [reply classes/phing/listener/DefaultLogger.php]: Would you mind adding a test for this method, please.

File: classes/phing/listener/DefaultLogger.php [hunk 2]
```diff
@@ -159,20 +159,20 @@ public function buildStarted(BuildEvent $event)
...
         }
+
         $msg .= PHP_EOL . "Total time: " . self::formatTime(Phing::currentTimeMillis() - $this->startTime) . PHP_EOL;
 
-        if ($error === null) {
-            $this->printMessage($msg, $this->out, Project::MSG_VERBOSE);
-        } else {
-            $this->printMessage($msg, $this->err, Project::MSG_ERR);
-        }
+        ($error === null)
```
Comment (siad007) [inline classes/phing/listener/DefaultLogger.php]: Please remove the unnecessary brackets.

**Comment by siad007** [comment]
@viniciuseneas thanks for your contribution

**Pull request merged**